docs/reference/modules/search.md

```diff
@@ -6,4 +6,91 @@ extends: '@apostrophecms/module'
 
 **Alias:** `apos.search`
 
-The seach
\ No newline at end of file
+The `@apostrophecms/search` module implements a sitewide search powered by the full-text search features of MongoDB. The module also provides the `@apostrophecms/search` page type to be used as a [parked page](/reference/modules/page.html#park). Like other page types, the module must be added to your `app.js` and project level `@apostrophecms/page/index.js` files.
+
+<AposCodeBlock>
+
+```js
+module.exports = {
+  options: {
+    park: [
+      {
+        slug: '/search',
+        parkedId: 'search',
+        title: 'Search',
+        type: '@apostrophecms/search'
+      }
+    ]
+  }
+};
+
+```
+<template v-slot:caption>
+modules/@apostrophecms/page/index.js
+</template>
+</AposCodeBlock>
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+`perPage` | Integer | Search results per page. Defaults to 10. |
+[`types`](#types) | Array | An array of page and piece type names to be searched. |
+[`filters`](#filters) | Array | An array of filters offered to the user to refine results. |
+
+### Types
+
+The `types` option takes an array of page-type and piece-type document names that will be included within the search results. By default, all page and piece docs are searched unless they have an option of `seachable: false`. Adding an array will restrict any search to just those types of pages and pieces, even if a module has an option of `searchable: true`.
```

Comment:
typo "seachable"

"unless that piece type module has an option of searchable: false"

I'm not sure if you can lock down to specific page types... maybe you can... or maybe you can just list `@apostrophecms/page` to include all page types, or not. I'm not sure offhand

Reply:
I misread the code comments - you are correct. Changed.